view/frontend/templates/product.phtml

```diff
@@ -130,7 +133,10 @@ $category = $this->getNostoCategory();
             <span class="alternate_image_url"><?php echo $this->escapeHtml($alternateImageUrl); ?></span>
         <?php endforeach; ?>
     </div>
-<?php if ($category) : ?>
-    <div class="nosto_category"
-         style="display:none"><?php echo $this->escapeHtml($category); ?></div>
-<?php endif; ?>
\ No newline at end of file
+    <?php if ($category) : ?>
```

Comment:
Leave this outside of the product check. It's possible to have active category without the product or with "invalidated" product

Reply:
Ok